### ISSUE TITLE
data race on write to []Blockchain

### ISSUE THREAD

**Issue by nosequeldeebee** [mycoralhealth/blockchain-tutorial#6]
in `/networking` tutorial:

https://gist.github.com/KosToZyB/5a549976daf7a3872e084ee7f57f6410

**Comment by kveselkov** [comment]
pull request to fix it: https://github.com/mycoralhealth/blockchain-tutorial/pull/7

**Comment by kveselkov** [comment]
@nosequeldeebee data race left, you need to add a mutex to function [replaceChain](https://github.com/mycoralhealth/blockchain-tutorial/blob/788919a998c5eb0ca33aa82a690964595c344f4c/networking/main.go#L136)
For example:
```
func replaceChain(newBlocks []Block) {
	mutex.Lock()
	if len(newBlocks) > len(Blockchain) {
		Blockchain = newBlocks
	}
	mutex.Unlock()
}
```

**Comment by kveselkov** [comment]
@nosequeldeebee  Add new PR: https://github.com/mycoralhealth/blockchain-tutorial/pull/8

**Comment by nosequeldeebee** [comment]
fixed by #8
